OJP/OJP_JourneySupport.xsd

```diff
@@ -600,12 +600,12 @@
 			<xs:documentation>parameter to control the number of TRIP results before/after a point in time. May NOT be used when departure time at origin AND arrival time at destination are set</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="NumberOfResultsBefore" type="xs:nonNegativeInteger">
+			<xs:element name="NumberOfResultsBefore" type="xs:nonNegativeInteger" minOccurs="0">
```

Comment:
I would rather have this with a default.

Reply:
@skinkie  better now?

Reply:
@ue71603 I don't know the details, but with a default, couldn't you just omit the minOccurs?

Reply:
doesn't work with minOccurs omitted (as the test says)

Reply:
Ok, then minOccurs is fine for me.